pyproject.toml

```diff
@@ -57,6 +55,8 @@ dependencies = [
     "packaging>=24",
     "pandas>=2.3",
     "pandera>=0.26",
+    "pillow<12",
```

Comment:
Needed to get around a dependency issue that really lives in `ferc-xbrl-extractor` See [this PR](https://github.com/catalyst-cooperative/ferc-xbrl-extractor/pull/382).